content/en/docs/languages/python/propagation.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Propagation
+description: Context propagation for the Python SDK
+aliases: [api/propagation]
```

Comment:
This is a new page, we shouldn't need an alias. Here's a followup PR to clean this up:

- #3887